Orange/widgets/regression/owlinearregression.py

```diff
@@ -45,30 +42,22 @@ class OWLinearRegression(OWProvidesLearner, widget.OWWidget):
                         range(20, 100, 5),
                         range(100, 1001, 100)))
 
-    def __init__(self):
-        super().__init__()
-
-        self.data = None
-        self.preprocessors = None
-
-        box = gui.widgetBox(self.controlArea, "Learner/Predictor Name")
-        gui.lineEdit(box, self, "learner_name")
-
+    def add_main_layout(self):
         box = gui.widgetBox(self.controlArea, "Regularization",
                             orientation="horizontal")
         gui.radioButtons(
-            box, self, "reg_type",
-            btnLabels=["No regularization", "Ridge regression (L2)",
-                       "Lasso regression (L1)", "Elastic net regression"],
-            callback=self._reg_type_changed)
+                box, self, "reg_type",
+                btnLabels=["No regularization", "Ridge regression (L2)",
+                           "Lasso regression (L1)", "Elastic net regression"],
+                callback=self._reg_type_changed)
 
         gui.separator(box, 20, 20)
         self.alpha_box = box2 = gui.widgetBox(box, margin=0)
         gui.widgetLabel(box2, "Regularization strength")
         self.alpha_slider = gui.hSlider(
-            box2, self, "alpha_index",
-            minValue=0, maxValue=len(self.alphas) - 1,
-            callback=self._alpha_changed, createLabel=False)
+                box2, self, "alpha_index",
+                minValue=0, maxValue=len(self.alphas) - 1,
+                callback=self._alpha_changed, createLabel=False)
```

Comment:
I agree. I hate it, too, when PyCharm does that, but haven't investigated whether it can be changed, so I always formatted it manually. (Thanks, @kernc.)

Double indentation is sometimes appropriate. Not here, though.